allennlp_models/modelcards/tagging-fine-grained-crf-tagger.json

```diff
@@ -0,0 +1,7 @@
+
+
+{
+    "name": "tagging-fine-grained-crf-tagger",
```

Comment:
Looking at this, I don't see why the "name" has to include all of this information.  If we rename this to `registered_model_name`, also, then it's more clear that you don't need to do this.

Oh, I see, below you are wanting to use this as the primary key for storing these objects.  Seems better to just not conflate the registered model name and the model card id.  Maybe add a new field for `registered_model_name`, or just `model_name`, keep the `name` as it is (or rename it to `id`, or something), and then you don't have to add all of the new calls to `Model.register`?